src/components/FormPills/FormPills.test.ts

```diff
@@ -266,4 +266,28 @@ describe('paste', () => {
 
         expect(modelValue.concat).not.toBeCalled();
     });
+
+    it('adds clipboard to input if data is invalid', () => {
```

Comment:
Still missing a lot of tests. Check [this one](https://github.com/owowagency/bootstrap-vue/blob/144c405a0bafbc88d3a9870dc1877ed4b4b8a7a2/src/components/FormDropdownSearch/FormDropdownSearch.test.ts) on how you could test computed properties.